requirements.txt

```diff
@@ -0,0 +1,11 @@
+attrs>=19.1.0
+black>=19.3b0
+boto3>=1.9.176
+cs>=2.7
```

Comment:
Since this is a module, requirements such as `cs` should go in setup.py, in `install-requires`

Reply:
Fixed in a66acf9.